index.html

```diff
@@ -0,0 +1,36 @@
+      
+<h2 >"Dr.Norman Borlaung Tribute"</h2>
+<div class="top-section">
+<p><span>Norman Ernest Borlaug,</span> (born March 25, 1914, near Saude, Iowa, U.S.—died September 12, 2009, Dallas, Texas), American agricultural scientist, plant pathologist, and winner of the Nobel Prize for Peace in 1970. Known as the “Father of the Green Revolution,” Borlaug helped lay
+the groundwork for agricultural technological advances that alleviated world hunger.
+</p>
+/div>     
+<div id="img-div">
+      
+<div id="img-caption">The Indian Vectory leader</div>
+      
+</div>
+<div class="middle-section"><p>
```

Comment:
Just as I suggested above I think it would be best practice to use the section tag at this point instead of div tag for best practice of html sematic tags.